apps/graphql/src/apps/itinerary/dataloaders/Itineraries.js

```diff
@@ -71,11 +76,41 @@ const addReturnSearchQueryParams = inboundDate => {
   };
 };
 
+const parseSlugsToIds = async (
+  location: SearchLocation,
+): Promise<SearchLocation> => {
+  if (!location.slugs) return location;
+
+  const results: $ReadOnlyArray<ApiResponseLocation> = await Promise.all(
+    location.slugs.map(async slug => fetch(`/locations/slug?term=${slug}`)),
```

Comment:
this async keyword is not necessary, since fetch already returns a promise

Reply:
Done!